tests/vodataservice/vodataservice_models_test.py

```diff
@@ -0,0 +1,370 @@
+"""Tests for VODataService models
+
+# TODO: This is an incomplete spec, covering only elements needed for VOSITables
```

Comment:
Should there be a issue/ticket about this?

Reply:
Added issue https://github.com/spacetelescope/vo-models/issues/17